src/bun.js/node/node_os.zig

```diff
@@ -173,7 +167,7 @@ pub const Os = struct {
                 } else if (std.mem.startsWith(u8, line, key_model_name)) {
                     // If this is the model name, extract it and store on the current cpu
                     const model_name = line[key_model_name.len..];
-                    slice[cpu_index].model = JSC.ZigString.init(model_name);
+                    slice[cpu_index].model = try JSC.ZigString.init(model_name).clone(allocator);
```

Comment:
Given that that line is just a slice into a local buffer, I changed this to a defensive copy.